cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -92,3 +109,18 @@ func parse(args []string) (*kong.Context, error) {
 
 	return parser.Parse(args)
 }
+
+func createLogger() *logrus.Logger {
```

Comment:
I wonder if we should try to log to `os.Stderr` by default (to make sure only the AWS credentials go to `os.Stdout`) and make file output optional?

Reply:
Per conversation with Roy, since this is all executed via scripting there needed to be a durable place for us to put errors so a human could inspect it after the fact. However, I do think it makes sense to try to log to both a file and standard error.